### ISSUE TITLE
UI for turning on overlay layer groups

### ISSUE THREAD

**Issue by chriswhong** [NYCPlanning/labs-nyc-factfinder#247]
This will be a zola-like layer palette where the user can turn toggle different layergroups for choropleths and subway layers

**Comment by chriswhong** [comment]
Blocked by #21 (Should happen after "select by variable" story, as the UI will be dependent on what we do in #21)

**Comment by andycochran** [comment]
Closing this. UI for "Advanced Settings" was resolved in `cw-choropleths`. When we work on #21 (`21-select-by-variable` or not), we won't need this separate story.
